pkgs/development/python-modules/expandvars/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "expandvars";
+  version = "0.9.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-ag54IeVf8atE+MoItQ0rLQHAdqm8l52pQEZ8t/EFxWU=";
+  };
```

Comment:
It's up to you, but you can feel free to use fetchFromGitHub to get the tests as well and run them. It's not as big of a deal here IMO as this package has few dependencies that can break it but I thought I'd mention it as an option.

Reply:
Good to know! For now I prefer to keep it this way since I expect depending directly on PyPi to be more resilient to changes, and we probably won't see many new dependencies appearing any time soon.